src/katgpucbf/send.py

```diff
@@ -45,6 +45,9 @@ class DescriptorSender:
     first_interval
         Delay (in seconds) immediately after starting. If not specified, it
         defaults to `interval`.
+    send_all_descriptors
```

Comment:
I'm not totally happy with the clarity of this name. How about `all_substreams`?